pyface/ui/qt4/console/console_widget.py

```diff
@@ -1336,10 +1336,12 @@ def _event_filter_console_keypress(self, event):
                     self._continuation_prompt
                 ):
                     self._control.moveCursor(
-                        QtGui.QTextCursor.MoveOperation.PreviousBlock, mode=anchormode
+                        QtGui.QTextCursor.MoveOperation.PreviousBlock,
+                        mode=anchormode,
```

Comment:
This change is purely cosmetic, right? (Just checking I haven't missed anything.)

And it's definitely true that `moveCursor` is happy to accept the `mode` keyword argument, even though `movePosition` apparently isn't?

Reply:
Yes, purely cosmetic - I touched it because I was checking to see if it also needed fixing (it doesn't).